db/data.js

```diff
@@ -320,9 +313,9 @@ const produce = [
   {
     foodName: "Cherry",
     imageURL:
-      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637899/Seasonal/057-organic-food_pikcaf.png",
+      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1666175319/Seasonal/006-cherries_hkayof.png",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed feugiat euismod augue, ac fringilla diam pharetra id. In porttitor convallis nisi vel finibus. Pellentesque at diam venenatis, ornare mi eget, vestibulum arcu. Vivamus consectetur, erat eu semper dapibus, dolor sem suscipit mi, ut dignissim enim nibh vel risus. Sed aliquet, sapien eu congue gravida, lorem metus bibendum mauris, quis maximus nibh erat eu erat. Cras vulputate purus magna, tincidunt porttitor massa vestibulum sed. Nam turpis massa, dictum eu accumsan in, posuere sit amet lorem. Phasellus est sapien, faucibus eget elit et, blandit consequat ex. Proin feugiat arcu a magna iaculis, nec pharetra orci porta. Proin vel orci ante. Aliquam viverra non ligula nec rhoncus. Ut et tincidunt elit. Phasellus non risus in mi vulputate venenatis vitae quis enim. Vestibulum quis dolor venenatis, lobortis diam in, faucibus turpis. Nullam quis pharetra nulla, vestibulum tincidunt ipsum. Donec non ultrices ligula. Quisque at interdum velit. Donec laoreet elit enim, a vulputate risus cursus eu. Duis et est in mi pulvinar rhoncus in ut lorem. Integer nec dolor et velit sollicitudin vehicula. Duis sapien justo, condimentum nec orci eu, dictum dapibus nisl. Praesent ultricies libero vel leo pulvinar molestie.",
+      "Cherries are a small, generally red fruit, which is at its height in the summer. There are hundreds of varieties, which can be sweet, sour, or somewhere between. Sweeter cherries are suitable for eating raw, and the more sour ones are generally used for cooking in things like tarts, crumbles, sauces and other desserts. When shopping for cherries, look for fruits with a deep colouring and firm flesh, but make sure that they are not too hard. Cherries can be grown in the UK climate, and are also suitable to be grown in containers. Traditionally, you would need more than one cherry tree in order for them to pollinate and bear fruit, but newer varieties of cherry trees are often self-pollinating.",
```

Comment:
Combination of plural "Cherries" but then singular references (e.g. at its height) in the first sentence?

Reply:
Thank you aquatic king.